include/swift/AST/DiagnosticsSema.def

```diff
@@ -2680,6 +2680,11 @@ ERROR(rethrows_without_throwing_parameter,none,
 ERROR(autoclosure_function_type,none,
       "@autoclosure attribute only applies to function types",
       ())
+
+ERROR(invalid_autoclosure_and_convention_attributes,none,
+      "@convention(%0) attribute is not allowed on @autoclosure type",
```

Comment:
Nit: `@convention(%0)` should be surrounded with single quotes (i.e. `'@convention(%0)'`). It's a bit inconsistent throughout this file, but we should be doing that. Also, 'types' not 'type'.

Reply:
Makes sense, thank's